src/rngs/adapter/reseeding.rs

```diff
@@ -279,11 +279,10 @@ where
 }
 
 
-#[cfg(all(unix, not(target_os = "emscripten")))]
+#[cfg(all(unix, feature = "std", not(target_os = "emscripten")))]
 mod fork {
```

Comment:
This requires `libc` and not `std`, right?

Reply:
`Once` requires `std`. `std` implies `libc` in the Cargo.toml

Reply:
Yes, it was a separate commit, so may be confusing.